utils/setup.py

```
@@ -356,6 +403,183 @@ def setup_project(process_handler, process_name):
+            )
+        if Instance_elem is not None and Instance_elem.text != app_name:
+            logger.info(
+                f"[{app_name}] Updating InstanceName in config.xml from {Instance_elem.text} to {app_name}"
+            )
+            Instance_elem.text = app_name
+            tree.write(config_file)
+    except Exception as e:
+        logger.error(f"[{app_name}] Failed to update existing config.xml: {e}")
+
+
+def setup_arr_instance(key, instance_name, instance, process_name):
```

Ruff (0.14.8)

Redefinition of unused `key` from line 1

(F811)